CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+# Introduction
+
+### Welcome dear contributor!
+
+First off, thank you for considering contributing to xcodeproj. It's people like you that make xcodeproj such a great tool.
+
+### Why reading the guidelines is important
+
+Following these guidelines helps to communicate that you respect the time of the developers managing and developing this open source project. In return, they should reciprocate that respect in addressing your issue, assessing changes, and helping you finalize your pull requests.
+
+### A good contribution
+
+xcodeproj is an open source project and we love to receive contributions from our community — you! There are many ways to contribute, from writing tutorials or blog posts, improving the documentation, submitting bug reports and feature requests or writing code which can be incorporated into xcodeproj itself.
+
+### A bad contribution
+
+Please, don't use the issue tracker for support questions. For that purpose use Stack Overflow tagging your threads with `#xcodeproj`.
+
+# Ground Rules
+
+* Ensure that changes in the project are tested.
+* Before merging, all the checks on GitHub should pass.
+* Create issues for any major changes and enhancements that you wish to make. Discuss things transparently and get community feedback.
+* Don't add any classes to the codebase unless absolutely needed.
+* Keep feature versions as small as possible, preferably one new feature per version.
+* Be welcoming to newcomers and encourage diverse new contributors from all backgrounds.
+* Comply the [code of conduct](https://github.com/swift-xcode/xcodeproj/blob/master/CODE_OF_CONDUCT.md) when you participate in the project.
+
+# Your First Contribution
+
+Unsure where to begin contributing to xcodeproj? You can start by looking through these easy and thread issues:
+
+- **difficulty:easy** - issues which should only require a few lines of code, and a test or two.
+- **type:thread** - issues which should be a bit more involved than beginner issues.
+
+Both issue lists are sorted by total number of comments. While not perfect, number of comments is a reasonable proxy for impact a given change will have.
+
+Working on your first Pull Request? You can learn how from this *free* series, [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github).
+
+At this point, you're ready to make your changes! Feel free to ask for help; everyone is a beginner at first :smile_cat:
+
+# Getting started
+
+1. Create your own fork of the code
```

Comment:
It's probably worth mentioning that you should run `swift package generate-xcodeproj` with Xcode 8 (will change with Swift 4 update) and then opening the xcode proj

Reply:
Good idea 😛, I'll update it.